group_test.go

```diff
@@ -0,0 +1,54 @@
+package opslevel_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/rocktavious/autopilot"
+)
+
+func TestGetGroup(t *testing.T) {
+	// Arrange
+	client := ATestClient(t, "group/get")
+	// Act
+	result, err := client.GetGroup("Z2lkOi8vb3BzbGV2ZWwvTmFtZXNwYWNlczo6R3JvdXAvMTI")
+	// Assert
+	fmt.Printf("%v", result)
+	autopilot.Ok(t, err)
+	autopilot.Equals(t, "test_group_1", result.Alias)
+	autopilot.Equals(t, nil, result.Parent.GroupId.Id)
+}
+
+func TestGetGroupWithAlias(t *testing.T) {
+	// Arrange
+	client := ATestClientAlt(t, "group/get", "group/get_with_alias")
+	// Act
+	result, err := client.GetGroupWithAlias("test_group_1")
+	// Assert
+	autopilot.Ok(t, err)
+	autopilot.Equals(t, "test_group_1", result.Alias)
+	autopilot.Equals(t, nil, result.Parent.GroupId.Id)
+}
+
+func TestListGroups(t *testing.T) {
+	// Arrange
+	client := ATestClient(t, "group/list")
+	// Act
+	result, err := client.ListGroups()
+	// Assert
+	autopilot.Ok(t, err)
+	autopilot.Equals(t, 2, len(result))
+	autopilot.Equals(t, "test_group_2", result[0].Alias)
+	autopilot.Equals(t, "test_group_1", result[1].Alias)
+}
+
+func TestListGroupsWithParent(t *testing.T) {
+	// Arrange
+	client := ATestClientAlt(t, "group/list", "group/list_with_parent")
+	// Act
+	result, err := client.ListGroupsWithParent("test_group_1")
+	// Assert
+	autopilot.Ok(t, err)
+	autopilot.Equals(t, 2, len(result))
```

Comment:
Copied this over from service tests but this is the only part that doesn't make sense to me.  I know we're mocking the request and response but I would expect the response to only have 1 item in the result.  I understand if it's for simplicity though so we don't have to have a different response per request.